scripts/3_find_max_intensity_diffusion_path.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import sys
+
+sys.path.append(r"C:\Users\harsh\quactography")
+
+from quactography.graph.undirected_graph import Graph
+from quactography.adj_matrix.io import load_graph
+from quactography.hamiltonian.hamiltonian_qubit_edge import Hamiltonian_qubit_edge
+from quactography.hamiltonian.hamiltonian_qubit_node import Hamiltonian_qubit_node
+from quactography.solver.qaoa_solver_qu_edge import multiprocess_qaoa_solver_edge
+from quactography.solver.qaoa_solver_qu_node import multiprocess_qaoa_solver_node
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter
+    )
+    p.add_argument(
+        "in_graph",
+        help="Adjacency matrix which graph we want path that maximizes weights in graph, npz file",
+        type=str,
+    )
+    p.add_argument("starting_node", help="Starting node of the graph", type=int)
```

Comment:
If you want to be consistant with scilpy, always put the help as the last argument (so after "type" in this case). Also, finish your sentences with a dot.